_sass/common.scss

```
@@ -281,6 +209,14 @@ header {
         pointer-events: none;
       }
 
+      &_open {
+        overflow: visible;
+
+        .selector-list {
+          pointer-events: all;
+        }
```

можно переворачивать галочку в селекте думы
```suggestion
        }
        &::after {
          transform: rotate(180deg);
        }
```

---

В макетах галочка не поворачивается

---

очень жаль, а в селекте карточки она переворачивается! хочется консистенции!